lib/ansible/modules/cloud/amazon/ec2_group.py

```diff
@@ -276,7 +276,13 @@ def main():
     # find the group if present
     group = None
     groups = {}
-    for curGroup in ec2.get_all_security_groups():
+
+    try:
+        security_groups = ec2.get_all_security_groups()
+    except Exception as e:
```

Comment:
@alexlo03 only inadvertently, I somehow switched from thinking about boto to boto3 exceptions. The same guidelines have boto exception handling - not sure if `boto.exceptions.BotoServerError` will suffice for timeouts or not.

http://boto.cloudhackers.com/en/latest/ref/boto.html#module-boto.exception

Given that we use BotoClientError with boto3, chances are same will work with boto